modules/ROOT/pages/_partials/commons/common-query-n1ql-mobile.adoc

```diff
@@ -766,7 +766,7 @@ SELECT db.*
   FROM db
   WHERE contact.name = "daniel"
 
-SELECT db.contact.address.city
+SELECT collection.contact.address.city
   FROM scope.collection
   WHERE contact.name = "daniel"
```

Comment:
Sorry, I missed thus. For consistency,

```
WHERE collection.contact.name = 'daniel'
```

Reply:
Can you add another example without `collection.` as well to avoid the impression that the `collection.` is required?

```
SELECT contact.address.city
FROM scope.collection
WHERE contact.name = "daniel"
```